### PR TITLE
feat: number converter improvements

### PR DESCRIPTION
**Pull request by andrewlew1s**
This is to address #56 

So far we can now mix and match bases to convert to and from! 

Screen Shot 2020-02-24 at 5 24 52 PM
Screen Shot 2020-02-24 at 5 25 03 PM

### CODE REVIEW

**Comment by andrewlew1s** [comment]
Tried to update the package version as last time:

**Comment by michaelfitzhavey** [comment]
@andrewlew1s did you check the version on npm? Since this package was deprecated this link takes me to a 404: https://www.npmjs.com/package/@holistic-web/eslint-config-toolbox

Refer to issue: #78 and replace `eslint-config-toolbox` with `"@holistic-web/eslint-config": "^0.1.2"`.

Or just run `git pull origin master` as these changes have been on the master branch for a while 😄

File: tools/number-converter/src/NumberConverter.vue [hunk 1]
```diff
@@ -15,9 +15,7 @@ Enter a number to convert:
 			:message="errorMessage"/>
 
 		<b-form-input
-			number
```
Comment (michaelfitzhavey) [inline tools/number-converter/src/NumberConverter.vue]: why are we removing the `number` prop?

File: tools/number-converter/src/NumberConverter.vue [hunk 2]
```diff
@@ -71,8 +81,16 @@ export default {
 			errorMessage: null,
 			inputNumber: null,
 			result: null,
-			base: 0,
-			baseOptions: [
+			fromBase: 0,
+			toBase: 0,
+			fromBaseOptions: [
```
Comment (michaelfitzhavey) [inline tools/number-converter/src/NumberConverter.vue]: this seems like duplication, can we not store both `to` and `from` in a single `baseOptions` object?

File: tools/number-converter/src/NumberConverter.vue [hunk 3]
```diff
@@ -110,16 +132,62 @@ export default {
 				this.errorMessage = err.message;
 			}
 		},
-		convertToBin(number) {
-			return number.toString(2);
+		convertToDec(number, fromBase) {
```
Comment (michaelfitzhavey) [inline tools/number-converter/src/NumberConverter.vue]: Lot's of repetition in `convertToDec`, `convertToBin`, etc... perhaps could use decimal as a 'middle' step? I.e. pseudocode for `onConvertClick`:

```
onConvertClick() {
    const value = convertFromBase(this.inputNumber, this.fromBase);
    const converted = convertToBase(value, this.toBase);
}
```

where `convertFromBase` and `convertToBase` are focused on converting numbers from base 10 to and from other bases respectively.

File: tools/number-converter/src/NumberConverter.vue (hunk 2 above)
Comment (andrewlew1s) [reply tools/number-converter/src/NumberConverter.vue]: yes!

Comment (michaelfitzhavey) [reply tools/number-converter/src/NumberConverter.vue]: let's do that 😄

Comment (andrewlew1s) [reply tools/number-converter/src/NumberConverter.vue]: done!

File: tools/number-converter/src/NumberConverter.vue (hunk 3 above)
Comment (andrewlew1s) [reply tools/number-converter/src/NumberConverter.vue]: I did this slightly differently, but with the same result of simplifying the code!

**Pull request merged**